.changeset/ninety-numbers-vanish.md

```diff
@@ -0,0 +1,30 @@
+---
+"@itwin/itwinui-css": major
+---
+
+Add `.iui-field` class to unify & centralize basic styling for `<button>`, `<input>`, `<textarea>`, and `<select>`.
```

Comment:
Should we mention class names (e.g. `.iui-input`) instead of HTML components (e.g. `<input>`)?

Because `<label class=".iui-input-flex-container">` or `<div class=".iui-input-flex-container">` would also require `.iui-field`, and not just `<input>`. Also, we apply `.iui-field` on `<div class="iui-select-button">` too and not exclusively on `<select>`.

Also, overall, it would be great if you could confirm if all user-facing changes have been adequately mentioned in changesets (either in this one or in separate changesets).

Reply:
Changeset updated to specify classes instead of elements.  Also went through each `.astro` file and made a point to mention other changes.

Reply:
Great, but not sure if the points are organized correctly. Because putting the points on separate lines in markdown would render as just one long paragraph.

<img width="668" alt="image" src="https://github.com/iTwin/iTwinUI/assets/45748283/b822f52c-15eb-417e-a036-39c754f1fd42">

So, maybe we need separate changeset files? Or maybe at least a bullet point list?

Reply:
Good catch. Formatting should be better now.